.github/workflows/checker-framework.yml

```diff
@@ -7,19 +7,19 @@ on:
   pull_request:
 
 jobs:
-  checker-framework:
+  analyse:
     strategy:
       fail-fast: false
       matrix:
         profile:
-          - checker-framework-nullness-optional-interning
-          - checker-framework-methods-resource-fenum
-          - checker-framework-lock-tainting
-          - checker-framework-index
-          - checker-framework-formatter
-          - checker-framework-signature-gui-units-init
-          - checker-framework-regex-property-key-compiler-message
-          - checker-framework-purity-value-returns
+          - checker-nullness-optional-interning
+          - checker-methods-resource-fenum
+          - checker-lock-tainting
+          - checker-index
+          - checker-formatter
+          - checker-signature-gui-units-init
+          - checker-regex-property-key-compiler-message
+          - checker-purity-value-returns
```

Comment:
Let's do this below, in job:
```yml

      - name: Execute checker framework
        env:
          MAVEN_OPTS: "-Xmx4g"
        run: groovy ./.ci/checker-framework.groovy checker-${{ matrix.profile }}

```

then we can just do: `- purity-value-returns`

Reply:
done